t/Test28-fregrid_stretched.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bats
+# test stretched grid 
+
+@test "Test stretched grid data lats 32.0 34.0 35.4" {
+
+  if [ ! -d "Test28" ] 
+  then
+  		mkdir Test28
+  fi
+
+  cd Test28
+   cp $top_srcdir/t/Test28-input/ocean_hgrid.nc . 
+   cp $top_srcdir/t/Test28-input/ocean_mosaic.nc .
+   cp $top_srcdir/t/Test28-input/topog.nc .
+
+#Make streetched grid 
+  run command make_hgrid \
+               --grid_type gnomonic_ed --do_schmidt \
+               --stretch_factor 2.5 \
+               --target_lon 262.4 \
+               --target_lat 32.0 \
+               --nlon 512 \
+               --grid_name C384_grid_32.0
+  [ "$status" -eq 0 ]
+
+  run command make_hgrid \
+               --grid_type gnomonic_ed --do_schmidt \
+               --stretch_factor 2.5 \
+               --target_lon 262.4 \
+               --target_lat 32.0 \
+               --nlon 512 \
+               --grid_name C384_grid_34.0
+  [ "$status" -eq 0 ]
+
+  run command make_hgrid \
+               --grid_type gnomonic_ed --do_schmidt \
+               --stretch_factor 2.5 \
+               --target_lon 262.4 \
+               --target_lat 35.4 \
+               --nlon 512 \
+               --grid_name C384_grid_35.4
+  [ "$status" -eq 0 ]
+
+
+#Create stretched grid mosaic
+  run command make_solo_mosaic \
+                --num_tiles 6 \
+                --dir ./ \
+                --mosaic_name C384_mosaic_32.0 --tile_file C384_grid_32.0.tile1.nc,C384_grid_32.0.tile2.nc,C384_grid_32.0.tile3.nc,C384_grid_32.0.tile4.nc,C384_grid_32.0.tile5.nc,C384_grid_32.0.tile6.nc
+  [ "$status" -eq 0 ]
+
+  run command make_solo_mosaic \
+                --num_tiles 6 \
+                --dir ./ \
+                --mosaic_name C384_mosaic_34.0 --tile_file C384_grid_34.0.tile1.nc,C384_grid_34.0.tile2.nc,C384_grid_34.0.tile3.nc,C384_grid_34.0.tile4.nc,C384_grid_34.0.tile5.nc,C384_grid_34.0.tile6.nc
+  [ "$status" -eq 0 ]
+
+  run command make_solo_mosaic \
+                --num_tiles 6 \
+                --dir ./ \
+                --mosaic_name C384_mosaic_35.4 --tile_file C384_grid_35.4.tile1.nc,C384_grid_35.4.tile2.nc,C384_grid_35.4.tile3.nc,C384_grid_35.4.tile4.nc,C384_grid_35.4.tile5.nc,C384_grid_35.4.tile6.nc
+  [ "$status" -eq 0 ]
```

Comment:
I would also suggest to add the make_coupler_mosaic step with --verbose --check. Although it is not required by fregrid,  it will give us numbers specifying how good is the grid in terms of covering the globe. E.g.,

time mpirun -np 16 make_coupler_mosaic_parallel --atmos_mosaic C256_mosaic.nc --ocean_mosaic ocean_mosaic.nc --mosaic_name mosaic --ocean_topog ocean_topog.nc --verbose --check

Reply:
Yes, I was actually working on adding a separate test for make_coupler_mosaic but we can also added as a step within the same test script.

Reply:
would it be ok to check for --area_ratio_thresh 1.e-10 as below :
 run bash -c 'mpirun -np 16 make_coupler_mosaic_parallel --atmos_mosaic C256_mosaic_32.0.nc \
                --ocean_mosaic ocean_mosaic.nc \
                --mosaic_name mosaic \
                --ocean_topog ocean_topog.nc \
                --verbose --check --area_ratio_thresh 1.e-10'
  echo "$output"
  [ "$status" -eq 0 ]